index.html

```diff
@@ -0,0 +1,290 @@
+            <div class="logo-holder">
+            <a href="#" class="my-logo"><img src="images/MyLogo.png " alt="my-logo"> </a>
+            </div>
+          <nav class="hamburger">
+          <a href="#"><i class="fa fa-bars"></i></a>
+          <ul class="ul-style">
+            <li>
+            <a class="portfolio" href="#"></a>
+          </li>
+          
+        </ul>       
+        </nav>
+      </div>
+      </header>
+      <div class="mainsection">
+        <div class="maincontainer">
+         
+       
+        
+      <div class="section1">
+        <h1 class="text1">
+         
+          i'm Abel<br/>
+          Glad to see you!<br/></h1>
+        
+        <div class="text2">
+         <p> I'm a software developer! i can help you build a product, feature or website Look through someof my work and experience! if you like what you see and have you need coded, don't hesitate to contact me.</p>
+        </div>
+        
+        <div class="let">
+        <a class="text3" href="#">LET'S CONNECT</a>
+        
+      
+      </div>
+     
+        
+        
+        <div class=" social_links">
+          <ul>
+            <li>
+              <a href="#">
+                <img class="twit-icon" src="./images/twitter.png" alt="twit-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="git-icon" src="./images/github.png" alt="git-icon">
+              </a>
+            </li>
+            
+            <li>
+              <a href="#">
+                <img class="ln-icon" src="./images/Linkedin-icon.png" alt="ln-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="m-icon" src="./images/medium.png" alt="m-icon">
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img class="a-icon" src="./images/angellist.png" alt="a-icon">
+              </a>
+            </li>
+           </ul>
+          </div>
+          </div>
+           </div>
+        </div>
+    <section class="work">
+      <div class="card-work">
+        <div class="card-image">
+          <img src="./images/Snapshootortfolio.jpg" alt="card-work">
+        </div>
+        <div class="left-block">
+          <h2 class="h2">Tonic</h2> 
+          <div class="frame1">
+            <p>CONOPY</p>
+            <img src="./images/Counter.jps" alt="">
+            <p class="p1">Back End Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2020</p>
+          </div>
+          <p class="primary1">
+            A daily selection of privately<br />personalized reads; 
+            no accounts or<br /> sign-ups required.
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action">
+            <p>See Project</p>
+          </div>
```

Comment:
- [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
![figma](https://user-images.githubusercontent.com/81727988/165932664-35948504-c2b3-4372-b10e-47c7cadc81e2.png)